plugin/session_view.py

```diff
@@ -37,47 +37,63 @@ class SessionView:
     _session_buffers = WeakValueDictionary()  # type: WeakValueDictionary[Tuple[int, int], SessionBuffer]
 
     def __init__(self, listener: AbstractViewListener, session: Session, uri: DocumentUri) -> None:
-        self.view = listener.view
```

Comment:
To avoid changing too much code, decided to use the "underscore" property in `__init__` and access through the getter elsewhere. This is maybe not super consistent but makes the diff smaller.